### PR TITLE
Add print USERNAME for Jenkins.

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Print username in format ```3 letters + space + the rest of letters```. If USERNAME is saved in Jenkins credentials it is not shown in a job (shown as set of ``` * ```) and is hard to track which user is used. 

Example:
for user ```osiotest2``` there will be shown ```User name printed in format: 3 first letters, space, the rest of letters.    osio test2```.

### CODE REVIEW

**Comment by ibuziuk** [comment]
@rhopp any security concerns?

File: functional-tests/devscripts/run_tests.sh [hunk 1]
```diff
@@ -54,6 +54,10 @@ else
 	TEST_SUITE="simpleTestSuite.xml"
 fi
 
+#This format allows us to see username even if it is placed in Jenkins credential store. 
+USERNAME_TO_PRINT="${USERNAME:0:3} ${USERNAME:3:${#USERNAME}}"
+echo "User name printed in format: 3 first letters, space, the rest of letters.    $USERNAME_TO_PRINT"
```
Comment (amisevsk) [inline functional-tests/devscripts/run_tests.sh]: Shouldn't this be 4 letters?

Comment (Katka92) [reply functional-tests/devscripts/run_tests.sh]: I don't know why but if you want, I'm ok with 4 letters too. :) Should I change it?

Comment (amisevsk) [reply functional-tests/devscripts/run_tests.sh]: Ah no, I was just confused, never mind.

`User name printed in format: 3 first letters, space, the rest of letters. osio test2.` would be `osi otest2` in this case, right?

Comment (Katka92) [reply functional-tests/devscripts/run_tests.sh]: Heh, yes, this is my typo. Sorry for confusion.

**Comment by rhopp** [comment]
By printing only username I don't think we are under any security risk...

**Pull request merged**